DEPENDENCIES.rst

```diff
@@ -0,0 +1,232 @@
+###########
+Cyclus Core
+###########
+
+The core of the Cyclus nuclear fuel cycle simulator from the
+University of Wisconsin - Madison is intended to be a simulation
+framework upon which to develop innovative fuel cycle simulations.
+
+To see user and developer documentation for this code, please visit
+the `Cyclus Homepage`_.
+
+.. contents:: Table of Contents
+   :depth: 2
+
+******************************
+Dependencies
+******************************
+
+In order to facilitate future compatibility with multiple platforms,
+Cyclus is built using `CMake`_. A full list of the Cyclus package
+dependencies is shown below:
+
+====================   ==================
+Package                Minimum Version
+====================   ==================
+`CMake`                2.8
+`boost`                1.46.1
+`libxml2`              2
+`libxml++`             2.36
+`python`               2.7 or 3.3+
+`sqlite3`              3.7.10
+`HDF5`                 1.8.4
+`Coin-Cbc`             2.5
+====================   ==================
+
+On some platforms, such as Ubuntu 14.04, the following are also necessary:
+
+====================   ==================
+Package                Minimum Version
+====================   ==================
+`g++`                  4.8.2
+`libblas-dev`          1.2
+`liblapack-dev`        3.5.0
+====================   ==================
+
+
+And a few optional dependencies:
+
+====================   ==================
+Package                Minimum Version
+====================   ==================
+doxygen (for docs)     1.7.6.1
+tcmalloc (for speed)   any?
+====================   ==================
+
+*Note that the Debian/Ubuntu package `libtcmalloc` is NOT discovered correctly
+by our build system.  Instead use `libgoogle-perftools-dev`.*
+
+******************************
+Installing Dependencies
+******************************
+
+Cyclus dependencies can either be installed via an operating system's package
+manager or via Conda.
+
+Via Conda
+---------
+
+*Note, at present, Conda dependency downloading is only supported for Linux*
+
+A Conda installation may be the most straightforward for a new developer. These
+instructions will be similar to those shown for Cyclus users `conda
+installation <http://fuelcycle.org/user/install.html>`_.
+
+#. Download the appropriate `miniconda installer <http://conda.pydata.org/miniconda.html>`_
+
+#. Install miniconda
+
+    .. code-block:: console
+
+        $ bash Miniconda-3.5.2-Linux-x86_64.sh -b -p ~/miniconda
+
+#. Add the following line to your ``.bashrc`` (``.bash_profile`` for Mac users):
+
+    ``export PATH-"${HOME}/miniconda/bin:${PATH}"``
```

Comment:
I didn’t know about that new feature of the conda installation, that’s cool!

@baaaaam and I discussed further yesterday. Maybe it’s more accurate to say that it’s possible to develop via Conda, but that if people plan to contribute back to the Cyclus community then we advise developing via git instead since git is the required workflow for contributions.  How does that sound?